test/fixtures/hybrid.template.json

```diff
@@ -54,8 +54,7 @@
                 {
                   "Effect": "Allow",
                   "Action": [
-                    "logs:FilterLogEvents",
-                    "logs:GetLogEvents"
+                    "logs:*"
```

Comment:
@zmully - can you describe more about this change? Was this temporarily for debugging purposes? Do you think we'll be able to scope this to least privilege again or do we need to keep it as *?